packages/react/src/components/CollapsibleNavbarItem/CollapsibleNavbarItem.stories.mdx

```diff
@@ -0,0 +1,50 @@
+import {ArgsTable, Source, Story, Canvas, Meta} from '@storybook/addon-docs';
```

Comment:
License header is missing

Reply:
Mdx files were exempted from adding the license header. If this is required, we will have to create an issue and update all the files.